src/amo/middleware/responseTime.js

```diff
@@ -0,0 +1,42 @@
+import config from 'config';
```

Comment:
It's too bad that the diff recognizes that the test was renamed, but not that this component was renamed. 😞